README.rst

```diff
@@ -29,28 +29,29 @@ Installation
 ------------
 
 
-Django-estimators is on PyPI, so just run: ::
+Run: 
+::
 
     pip install django-estimators
 
 
 Quick start
 -----------
 
-1. Add "estimators" to your INSTALLED_APPS django setting like this
+1. Add ``estimators`` to your INSTALLED_APPS.
 ::
 
     INSTALLED_APPS = [
         ...
         'estimators',
     ]
   
-2. To create the estimators table, run
+2. Create the estimators table.
 ::
 
     python manage.py migrate
 
-3. Run ``python manage.py shell`` and get create new models like so
+3. Create a new model. Run ``python manage.py shell``
```

Comment:
Create and save a new model.  Open the python interpreter ``python manage.py shell`` and run